kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -374,6 +385,13 @@ public static Map<String, Variant<?>> buildIpv6Settings(NetworkProperties props,
             logger.warn("Unexpected ip status received: \"{}\". Ignoring", ip6Status);
         }
 
+        Optional<Integer> mtu = props.getOpt(Integer.class, "net.interface.%s.config.ip6.mtu", deviceId);
+        logger.info("REMOVEME-- ip6mtu is {}", mtu);
+        //ipv6.mtu only supported in networkManager 1.40 and above
+        logger.info("--REMOVEME; {} > 1.40? {}", nmVersion, nmVersion.isGreaterEqualThan("1.40"));
+        if (nmVersion.isGreaterEqualThan("1.40")) {
+            mtu.ifPresent(value -> settings.put("mtu", new Variant<>(new UInt32(value))));
+        }
```

Comment:
For a better UX I would log a warning if and only if the mtu is set and the version is lower than 1.40. Something like:

```suggestion
        Optional<Integer> mtu = props.getOpt(Integer.class, "net.interface.%s.config.ip6.mtu", deviceId);
        if(mtu.isPresent()) {
          if (nmVersion.isGreaterEqualThan("1.40")) {
              settings.put("mtu", new Variant<>(new UInt32(mtu.get())));
          } else {
             logger.warn("NetworkManager version lower than 1.40, 'mtu' parameter ignored");
          }
       }
```

but I'm open to nicer suggestions